plugins/org.brainwy.liclipsetext.editor/plugin.xml

```diff
@@ -216,7 +216,7 @@
     <!-- Note: run the install.py to regenerate the actions contents! -->
     <extension point="org.eclipse.core.contenttype.contentTypes">
         <content-type base-type="org.eclipse.core.runtime.xml"
-            file-extensions="xml,xsd" file-names=".pydevproject,.project,.classpath"
+            file-extensions="xml,xsd" file-names=".pydevproject,.project,.classpath,.launch"
```

Comment:
I don't know Python and don't have it installed (on my Windows PC)

I guess this project README should hint clearly how to add a language support